inclusive-design/idrc
# feat: configure visual regression tests with BackstopJS

## Description

Adds local visual regression testing with BackstopJS.

TODO:

- [x] Run visual regression tests in CI.

## Steps to test

1. Build site.
2. Run `npm run start`.
3. In a new terminal window, run `npm run test:backstop`.

**Expected behavior:** Test results load.

## Additional information

Docs: https://github.com/garris/BackstopJS

## Related issues

Not applicable.
